aptos-move/aptos-gas/src/aptos_framework.rs

```diff
@@ -8,6 +8,7 @@ crate::natives::define_gas_parameters_for_natives!(GasParameters, "aptos_framewo
     [.account.create_address.base, "account.create_address.base", 300 * MUL],
     [.account.create_signer.base, "account.create_signer.base", 300 * MUL],
 
+    [.algebra.placeholder, { 8.. => "algebra.placeholder"}, 1],
```

Comment:
I assume we'll need to bump LATEST_GAS_FEATURE_VERSION

Reply:
Hmm... should also make sure we get rid of the placeholder before the next branch cut. This will be replaced by real gas parameters in subsequent PRs, right?

Reply:
@vgao1996 yes they will be replaced in the following PRs.

Reply:
@vgao1996, is the gas feature bumping done by this PR or by a future one?

And when is the next branch cut, because the subsequent PR might not get merged in time.

Reply:
And regarding removing the placeholder before the next branch cut, what if the next PR does not get merged in time?

Reply:
> And regarding removing the placeholder before the next branch cut, what if the next PR does not get merged in time?

That's exactly the point I was trying to made with my original comment: we must not let this happen. Either merge the next PR in time or put one up to get this removed.

Reply:
I see. Should we:

1. Submit a small PR that removes all gas parameters and charges 0 (temporarily for the natives)

2. Submit a small PR that uses the correctly named gas parameters (but not correctly gased). These would be a subset of all the parameters, but it would be okay to upgrade it later?